<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>1090</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WheatDetector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblSquares">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>930</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Площадь поля:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSrPlot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>660</y>
      <width>930</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cредняя плотность пшеницы на поле:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReturn">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>950</y>
      <width>520</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить новые данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblKol">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>740</y>
      <width>930</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Общее число колосьев на поле:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>120</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>730</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите поле, результаты которого Вы хотите увидеть</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>195</y>
      <width>460</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Форма поля, по введенным Вами координатам</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZagruzka">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>810</y>
      <width>391</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Экспортировать файл с разметкой колосьев пшеницы</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDownload">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>900</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузка успешно завершена</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
